sdk/identity/azure-identity/tests/test_auth_code.py

```diff
@@ -153,36 +150,3 @@ def send(request, **_):
     # should still default to the first tenant
     token = credential.get_token("scope")
     assert token.token == first_token
-
-
-def test_multitenant_authentication_not_allowed():
```

Comment:
Do you think there would be value in testing the case where `AZURE_IDENTITY_DISABLE_MULTITENANTAUTH` is set to True but a `tenant_id` is still provided?